src/plugin/elan.js

```diff
@@ -237,7 +237,7 @@ export default class ElanPlugin {
             const th = document.createElement('th');
             th.className = 'wavesurfer-tier-' + tier.id;
             th.textContent = tier.id;
-            th.style.width = this.params.tiers[tier.id];
+            th.style.width = true;
```

Comment:
Thanks for taking a look.  I agree that does not make a lot of sense, but that is the value that it takes under the existing behavior when `tiers` is specified.

This [snippet from the example](https://github.com/katspaugh/wavesurfer.js/blob/master/example/elan/app.js#L17-L24) sets the value under specific ids to true.
```
            WaveSurfer.elan.create({
                url: 'transcripts/001z.xml',
                container: '#annotations',
                tiers: {
                    Text: true,
                    Comments: true
                }
            }),
```

Then `this.params.tiers[tier.id]` ends up with the value `true`.

I was aiming to avoid a larger refactor of this plugin, and just replicate the existing behavior when `params.tiers` is provided.  Note that `tiers` gets filtered down in line 196 if that parameter exists, `indeces` is only populated for tiers that exist past that filter, and the line 240 in question is called for each value of `indeces`.

Another approach would be to just delete line 240.

Reply:
> Another approach would be to just delete line 240.

If it doesn't do anything, for sure, let's delete it.

Reply:
That works.  I just removed that line and confirmed the example Elan app still works (with and without the provided params.tiers).